doc/mgr/nfs.rst

```diff
@@ -447,6 +500,53 @@ where:
 
 ``<pseudo_path>`` is the pseudo root path (must be an absolute path).
 
+Enable QoS bandwidth control for export
+---------------------------------------
+
+.. code:: bash
+
+    $ nfs export qos enable bandwidth_control <cluster_id> <pseudo_path> [--combined-rw-bw-ctrl] [--max_export_write_bw <value>] [--max_export_read_bw <value>] [--max_client_write_bw <value>] [--max_client_read_bw <value>] [--max_export_combined_bw <value>] [--max_client_combined_bw <value>]
+
+This command will enable/update QoS bandwidth control for export. It will create QOS_BLOCK in export block, where
+
+``<cluster_id>`` is the NFS Ganesha cluster ID.
+
+``<pseudo_path>`` is the pseudo root path (must be an absolute path).
+
+``<combined-rw-bw-ctrl>`` will enable combined read and write bandwidth. If its True, then only max_export_combined_bw and max_client_combined_bw paramaters are allowed based on qos_type.
+
+``<max_export_write_bw>`` is the maximum bandwidth that can be used for export write.
+
+``<max_export_read_bw>`` is the maximum bandwidth that can be used for export read.
+
+``<max_client_write_bw>`` is the maximum bandwidth that client can use for write.
+
+``<max_client_read_bw>`` is the maximum bandwidth that client can use for read.
+
+``<max_export_combined_bw>`` is the maximum combined read/write bandwidth for export.
+
+``<max_client_combined_bw>`` is the maximum combined read/write bandwidth for client.
+
+Disable QoS bandwidth control for export
+----------------------------------------
+
+.. code:: bash
+
+   $ nfs export qos disable bandwidth_control <cluster_id> <pseudo_path>
+
+This will disable QoS bandwidth control for export. Export can use unlimited I/O bandwidth after disabling export qos i.e. it will not follow cluster level values.
+
+.. note:: To use cluster level qos bandwidth control values for export again, we can use ``nfs export apply <cluster_id> `` command, with export block not having QOS_BLOCK.
```

Comment:
s/<cluster_id> ``/<cluster_id>``/

The backticks have to be immediately adjacent to the pointy bracket (the greater-than sign), or Sphinx will be confused and it'll throw this ``Inline literal start-string without end-string.`` error.